com.unity.ml-agents/Runtime/Actuators/ActuatorManager.cs

```diff
@@ -157,9 +158,11 @@ public ActionSpec GetCombinedActionSpec()
         /// actions for the IActuators in this list.</param>
         public void UpdateActions(ActionBuffers actions)
         {
+            Profiler.BeginSample("ActuatorManager.UpdateActions");
```

Comment:
If you want profiling on these, you can also use
```
using (TimerStack.Instance.Scoped("ActuatorManager.UpdateActions"))
{
   ...
}
```
(which adds Profiler samples too)